test/e2e/storage/persistent_volumes-local.go

```diff
@@ -120,7 +120,7 @@ const (
 	// provisioner daemonSetName name
 	daemonSetName = "local-volume-provisioner"
 	// provisioner default mount point folder
-	provisionerDefaultMountRoot = "/mnt-local-storage"
+	provisionerDefaultMountRoot = "/mnt/local-storage"
```

Comment:
Can you change the hostpath volume config to also use this variable?

Reply:
@msau42 Could you please point me to the line? Are you referring to this line?

        provisionerConfig.StorageClassConfig = map[string]MountConfig{
                config.scName: {
                        HostDir:  path.Join(hostBase, discoveryDir),  < ====== ?????
                        MountDir: provisionerDefaultMountRoot,
                },
        }

Reply:
https://github.com/sbezverk/kubernetes/blob/46eb099d223ed7034f9d579bdf9ec7a2d2223042/test/e2e/storage/persistent_volumes-local.go#L1074

Reply:
done